src/main/webapp/dev/js/common/assert.es6

```diff
@@ -0,0 +1,19 @@
+class AssertionFailedError extends Error {
+}
+
+function assert(condition, message) {
+    const msg = message === undefined || message === null ? 'Assertion Failed' : message;
+    if (!condition) {
+        throw new AssertionFailedError(msg);
```

Comment:
I don't see them being used in JavaScript much... I guess because they don't handle `null`, empty string, and other such values interpreted as false when used as a condition?